hw/ip/dma/data/dma.hjson

```diff
@@ -558,11 +559,21 @@
                 '''
         }
         { bits: "2"
+          name: "chunk_done"
+          resval: 0x0
+          hwaccess: "hrw"
```

Comment:
As per comments on other reviews, are you(pl) happy with breaking backwards compatibility by inserting a new status bit rather than keeping the existing bits in their previous locations?

Reply:
It was changed to keep the existing bit positions and add the new one at the end.